src/main/java/com/aca/homework/week15/sql/company-sql.txt

```diff
@@ -0,0 +1,20 @@
+There are a lot of companies and users that can not exist outside of the company.
```

Comment:
because users can not exist outside of the company you could create company and company_user only.

please git it a try with 2 tables only